## STDOUT ##

[[92mINFO[0m] SC2 4.9.0 Ïö© Ìå®Ïπò Ï†ÅÏö©
Ìå®Ïπò sc2.ids.ability_id
Ìå®Ïπò sc2.game_state
Ìå®Ïπò sc2.bot_ai
Ìå®Ïπò sc2.client
Ìå®Ïπò sc2.controller
Ìå®Ïπò sc2.main
INFO:root:Failed to import cuda module: No module named 'caffe2.python.caffe2_pybind11_state_gpu'
INFO:root:Failed to import AMD hip module: No module named 'caffe2.python.caffe2_pybind11_state_hip'
WARNING:root:This caffe2 python run does not have GPU support. Will run in CPU only mode.
WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/bot.py:39: The name tf.reset_default_graph is deprecated. Please use tf.compat.v1.reset_default_graph instead.

WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/bot.py:40: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/bot.py:42: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/rl/agents/armyagent3.py:27: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/rl/agents/armyagent3.py:27: The name tf.AUTO_REUSE is deprecated. Please use tf.compat.v1.AUTO_REUSE instead.

WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/rl/agents/armyagent3.py:32: conv2d (from tensorflow.python.layers.convolutional) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.keras.layers.Conv2D` instead.
WARNING:tensorflow:From /home/hspark8312/anaconda3/envs/sc2/lib/python3.6/site-packages/tensorflow/python/ops/init_ops.py:1251: calling VarianceScaling.__init__ (from tensorflow.python.ops.init_ops) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Call initializer instance with the dtype argument instead of passing it to the constructor
WARNING:tensorflow:Entity <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x7f028b9febe0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x7f028b9febe0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x7f028b9febe0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x7f028b9febe0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/rl/agents/armyagent3.py:44: flatten (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.flatten instead.
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7f028b9febe0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7f028b9febe0>>: AttributeError: module 'gast' has no attribute 'Num'
WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/rl/agents/armyagent3.py:37: dense (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dense instead.
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f028b9febe0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f028b9febe0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f028b9febe0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f028b9febe0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f027349edd8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f027349edd8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f02701116d8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f02701116d8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/rl/ppo.py:56: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/rl/ppo.py:70: The name tf.train.AdamOptimizer is deprecated. Please use tf.compat.v1.train.AdamOptimizer instead.

WARNING:tensorflow:From /home/hspark8312/anaconda3/envs/sc2/lib/python3.6/site-packages/tensorflow/python/ops/math_grad.py:1250: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/rl/ppo.py:33: The name tf.summary.FileWriter is deprecated. Please use tf.compat.v1.summary.FileWriter instead.

WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/rl/ppo.py:80: The name tf.summary.scalar is deprecated. Please use tf.compat.v1.summary.scalar instead.

WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/rl/ppo.py:88: The name tf.summary.histogram is deprecated. Please use tf.compat.v1.summary.histogram instead.

WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/rl/ppo.py:35: The name tf.summary.merge_all is deprecated. Please use tf.compat.v1.summary.merge_all instead.

WARNING:tensorflow:From /mnt/d/sc2minigame-eval-2020-01-09/bots/npc/bot.py:54: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.

WARNING:tensorflow:From /home/hspark8312/anaconda3/envs/sc2/lib/python3.6/site-packages/tensorflow/python/training/saver.py:1276: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
INFO:tensorflow:Restoring parameters from bots/npc/models/DropArmyAgent/model.ckpt
Model restored to global
INFO:sc2.protocol:Client status changed to Status.launched (was None)
INFO:sc2.protocol:Client status changed to Status.launched (was None)
INFO:sc2_patch.sc2_patch_490.controller:Creating new game
INFO:sc2_patch.sc2_patch_490.controller:Map:     NCFellowship-2019_m1_v7
INFO:sc2_patch.sc2_patch_490.controller:Players: Bot(Race.Terran, <bots.transfer.bot.Bot object at 0x7f02a1606860>), Bot(Race.Terran, <bots.npc.bot.Bot object at 0x7f029351cbe0>)
INFO:sc2.protocol:Client status changed to Status.init_game (was Status.launched)
INFO:sc2.protocol:Client status changed to Status.in_game (was None)
INFO:root:Player 1 - Bot(Race.Terran, <bots.transfer.bot.Bot object at 0x7f02a1606860>)
TEST: 1, <bots.transfer.bot.Bot object at 0x7f02a1606860>
INFO:sc2.protocol:Client status changed to Status.in_game (was None)
INFO:root:Player 2 - Bot(Race.Terran, <bots.npc.bot.Bot object at 0x7f029351cbe0>)
TEST: 2, <bots.npc.bot.Bot object at 0x7f029351cbe0>
load model: bots/transfer/model/model.pt
INFO:sc2.protocol:Client status changed to Status.ended (was Status.in_game)
INFO:root:Result for player 1 - Bot(Race.Terran, <bots.transfer.bot.Bot object at 0x7f02a1606860>): Defeat
INFO:sc2.protocol:Client status changed to Status.ended (was Status.in_game)
INFO:root:Result for player 2 - Bot(Race.Terran, <bots.npc.bot.Bot object at 0x7f029351cbe0>): Victory
INFO:sc2.client:Saved replay to outs/2020-02-06T11-52-22/replays/transfer-npc/transfer-npc-26.SC2Replay
INFO:sc2.protocol:Client status changed to Status.launched (was Status.ended)
INFO:sc2.protocol:Client status changed to Status.quit (was Status.launched)
INFO:sc2.sc2process:kill_switch: Process cleanup
INFO:sc2.sc2process:Cleaning up...
INFO:sc2.sc2process:Cleanup complete
INFO:sc2.sc2process:Cleaning up...
INFO:sc2.sc2process:Cleanup complete
INFO:sc2.protocol:Cannot receive: Connection already closed.
INFO:sc2.sc2process:kill_switch: Process cleanup
INFO:sc2.sc2process:Cleaning up...
INFO:sc2.sc2process:Cleanup complete
INFO:sc2.sc2process:Cleaning up...
INFO:sc2.sc2process:Cleanup complete


## STDERR ##

2020-02-10 14:26:00.711643: I tensorflow/core/platform/cpu_feature_guard.cc:145] This TensorFlow binary is optimized with Intel(R) MKL-DNN to use the following CPU instructions in performance critical operations:  SSE4.1 SSE4.2 AVX AVX2 FMA
To enable them in non-MKL-DNN operations, rebuild TensorFlow with the appropriate compiler flags.
2020-02-10 14:26:00.723140: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3000000000 Hz
2020-02-10 14:26:00.724564: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x7fffc306a240 executing computations on platform Host. Devices:
2020-02-10 14:26:00.724589: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
OMP: Info #212: KMP_AFFINITY: decoding x2APIC ids.
OMP: Info #213: KMP_AFFINITY: x2APIC ids not unique - decoding legacy APIC ids.
OMP: Info #232: KMP_AFFINITY: legacy APIC ids not unique - parsing /proc/cpuinfo.
OMP: Info #148: KMP_AFFINITY: Affinity capable, using cpuinfo file
OMP: Info #154: KMP_AFFINITY: Initial OS proc set respected: 0-7
OMP: Info #156: KMP_AFFINITY: 8 available OS procs
OMP: Info #157: KMP_AFFINITY: Uniform topology
OMP: Info #179: KMP_AFFINITY: 1 packages x 8 cores/pkg x 1 threads/core (8 total cores)
OMP: Info #214: KMP_AFFINITY: OS proc to physical thread map:
OMP: Info #171: KMP_AFFINITY: OS proc 0 maps to package 0 core 0
OMP: Info #171: KMP_AFFINITY: OS proc 1 maps to package 0 core 1
OMP: Info #171: KMP_AFFINITY: OS proc 2 maps to package 0 core 2
OMP: Info #171: KMP_AFFINITY: OS proc 3 maps to package 0 core 3
OMP: Info #171: KMP_AFFINITY: OS proc 4 maps to package 0 core 4
OMP: Info #171: KMP_AFFINITY: OS proc 5 maps to package 0 core 5
OMP: Info #171: KMP_AFFINITY: OS proc 6 maps to package 0 core 6
OMP: Info #171: KMP_AFFINITY: OS proc 7 maps to package 0 core 7
OMP: Info #250: KMP_AFFINITY: pid 2893 tid 2893 thread 0 bound to OS proc set 0
2020-02-10 14:26:00.725454: I tensorflow/core/common_runtime/process_util.cc:115] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
2020-02-10 14:26:01.708662: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.
OMP: Info #250: KMP_AFFINITY: pid 2893 tid 2924 thread 1 bound to OS proc set 1
Version: B75689 (SC2.4.10)
Build: Aug 12 2019 17:16:57
Command Line: '"/home/hspark8312/StarCraftII/Versions/Base75689/SC2_x64" -listen 127.0.0.1 -port 19487 -displayMode 0 -dataDir /home/hspark8312/StarCraftII -tempDir /tmp/SC2_6wzivq9z'
Starting up...
Startup Phase 1 complete
Version: B75689 (SC2.4.10)
Build: Aug 12 2019 17:16:57
Command Line: '"/home/hspark8312/StarCraftII/Versions/Base75689/SC2_x64" -listen 127.0.0.1 -port 21511 -displayMode 0 -dataDir /home/hspark8312/StarCraftII -tempDir /tmp/SC2_pfsyge7x'
Starting up...
Startup Phase 1 complete
Startup Phase 2 complete
Creating stub renderer...
Listening on: 127.0.0.1:19487
Startup Phase 3 complete. Ready for commands.
Startup Phase 2 complete
Creating stub renderer...
ConnectHandler: Request from 127.0.0.1:57600 accepted
ReadyHandler: 127.0.0.1:57600 ready
Requesting to join a multiplayer game
Configuring interface options
Configure: raw interface enabled
Configure: feature layer interface enabled
Configure: score interface enabled
Configure: render interface disabled
Listening on: 127.0.0.1:21511
Startup Phase 3 complete. Ready for commands.
ConnectHandler: Request from 127.0.0.1:57601 accepted
ReadyHandler: 127.0.0.1:57601 ready
Requesting to join a multiplayer game
Configuring interface options
Configure: raw interface enabled
Configure: feature layer interface enabled
Configure: score interface enabled
Configure: render interface disabled
Create game with map path: NCFellowship-2019_m1_v7.SC2Map
Attempting to join a game with the map: NCFellowship-2019_m1_v7.SC2Map
Game has started.
Using default stable ids, none found at: /home/hspark8312/StarCraftII/stableid.json
Successfully loaded stable ids: GameData\stableid.json
Game has started.
Using default stable ids, none found at: /home/hspark8312/StarCraftII/stableid.json
Successfully loaded stable ids: GameData\stableid.json
Sending ResponseJoinGame
Sending ResponseJoinGame
OMP: Info #250: KMP_AFFINITY: pid 2893 tid 2925 thread 2 bound to OS proc set 2
RequestQuit command received.
Closing Application...
CloseHandler: 127.0.0.1:57601 disconnected
Terminate action already called.
/home/hspark8312/anaconda3/envs/sc2/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
  return f(*args, **kwds)
/home/hspark8312/anaconda3/envs/sc2/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
  return f(*args, **kwds)
/home/hspark8312/anaconda3/envs/sc2/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/hspark8312/anaconda3/envs/sc2/lib/python3.6/site-packages/past/builtins/misc.py:45: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  from imp import reload
/home/hspark8312/anaconda3/envs/sc2/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/hspark8312/anaconda3/envs/sc2/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 216, got 192
  return f(*args, **kwds)
/home/hspark8312/anaconda3/envs/sc2/lib/python3.6/importlib/_bootstrap.py:219: ImportWarning: can't resolve package from __spec__ or __package__, falling back on __name__ and __path__
  return f(*args, **kwds)
/home/hspark8312/anaconda3/envs/sc2/lib/python3.6/site-packages/tensorflow/python/keras/backend.py:5506: ResourceWarning: unclosed file <_io.TextIOWrapper name='/home/hspark8312/.keras/keras.json' mode='r' encoding='UTF-8'>
  _config = json.load(open(_config_path))
/home/hspark8312/anaconda3/envs/sc2/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
  return f(*args, **kwds)
/home/hspark8312/anaconda3/envs/sc2/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
  return f(*args, **kwds)
